src/app/components/footer/footer.component.html

```diff
@@ -1,8 +1,8 @@
 <footer class="bg-light text-center text-lg-start">
 
-    <div class="text-center p-3" style="background-color: #3c3c3b">
+    <div class="text-center p-3" style="background-color: #3c3c3b ; padding: 8px" >
```

Comment:
Let's add a CSS class here instead of the style attribute, something like `.footer`. Also try to use relative units like `em` or `pm` instead of `px`.